### ISSUE TITLE
Importing audio.wav along with dng frames with import dng menu dialogue...

### ISSUE THREAD

**Issue by 2blackbar** [ilia3101/MLV-App#262]
Would this be possible to add ? Im filming a lot with a phone and it spits DNGs with audio.wav, i cant import audio with frames tho and it would really speedup the workflow
Or maybe adding an option to pick our own wav file when importing dngs? 
Anything that would make this semiautomatic so when i film 40 clips i dont have to pick audio 40 times by browsing the folders with dngs...

**Comment by masc4ii** [comment]
You're right: more important is the request on raw2mlv side. MLVApp just implements a dialog which controls this little cmd app.
The biggest problem will be: for DNG+WAV there is no rule how video and audio is synced (as far as I know). In MLV, there is a rule, coming with the mlv architecture and internal structure. So you'll get bad sync in many cases, plus all input must come from the user (audio start delay (±), correct framerate, how to handle if video is longer than audio, or audio is longer than video, etc. ...). No idea if it is worth the effort - I'd recommend to sync audio+video in a NLE, where you can drag the audio to the correct position, and set the cut in / cut out.

**Comment by 2blackbar** [comment]
Yeh thats what i do at the moment just export video and drag all audio clips later to audio track... 
They just changed audio naming so all audio tracks are named audio.wav which made it harder to identify, ill try asking about it on raw2mlv github fork
